examples/radius_full.toml

```diff
@@ -0,0 +1,29 @@
+uri = "https://example.com" # URL to the Kanidm server
+verify_hostnames = true     # verify the hostname of the Kanidm server
+
+ca = "/data/ca.pem"          # Path to the kanidm caverify_ca = false # Strict CA verification
```

Comment:
We also support a ca_dir I think? 

It looks like these are the wronge names btw, I think it's:

```
radius_cert_path = "/data/cert.pem"
radius_key_path = "/data/key.pem"
radius_dh_path = "/data/dh"
radius_ca_path = "/data/cert.pem"
radius_ca_dir = "/data/ca/"
```